build-system/server/server.js

```diff
@@ -26,8 +26,8 @@ const isRunning = require('is-running');
 const log = require('fancy-log');
 const webserver = require('gulp-webserver');
 
-const host = process.env.SERVE_HOST;
-const port = process.env.SERVE_PORT;
+const host = process.env.SERVE_HOST || process.env.HOST;
```

Comment:
I think the changes on lines 29 and 30 are redundant, since the `build-system/tasks/serve.js` script is already taking care of passing along values from the HOST and PORT env vars

Reply:
If I'm following correctly, the tasks folder is only used by gulp, right? I modified the package.json `start-remote` script as well to be able to use `.env` vars as well to be able to run the app without gulp as it would in production, so added the alternate port/host here as well.